python/paddle/fluid/framework.py

```diff
@@ -642,7 +642,8 @@ def __init__(self,
         if type is None:
             raise ValueError(
                 "`type` to initilized an Operator can not be None.")
-        else:
+
+        if os.getenv("FLAGS_enable_callstack"):
```

Comment:
> Please don't disable this since http://newicafe.baidu.com/issue/DLTP-124/show?from=page

Sorry, i can't open this link. Is it a internal link?
Besides, is it possible to provide an option to disable the callstack detail? 
Those debug-level information will make it's hard to read the program.